README.md

```diff
@@ -14,17 +14,18 @@ This extension adds language support for [Lingua Franca](https://www.lf-lang.org
 * target syntax highlighting
 * code validation upon edit
 * target code validation upon build (error highlighting currently for C++ only)
-* user-triggered build (<kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>P</kbd>, then `Build Lingua Franca`) 
+* user-triggered build (<kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>P</kbd>, then `Lingua Franca: Build`) 
 
 ## Quick Start
  1. Install this plugin from the [VSCode
     Marketplace](https://marketplace.visualstudio.com/items?itemName=lf-lang.vscode-lingua-franca)/[Open VSX Registry](https://open-vsx.org/extension/lf-lang/vscode-lingua-franca)
     (in the command palette <kbd>Ctrl</kbd> + <kbd>P</kbd>, enter `ext install lf-lang.vscode-lingua-franca`)
  2. (Skip this step if you already have Lingua Franca projects that you'd like
     to work on.) Create a new Lingua Franca project by creating a `<My Project
-    Name>/src` folder and putting a file in it that has the `.lf` extension.
- 3. Open a Lingua Franca project (`File > Add Folder to Workspace...`). Open the
-    folder for the whole project (i.e., the folder containing the `src` folder).
+    Name>/src` folder and put a file in it that has the `.lf` extension.
+ 3. Open a Lingua Franca project (`File > Add Folder to Workspace...`). The Explorer (upper left in sidebar) should now show your project files. Open the `.lf` file you created.
```

Comment:
Step 3 and 2 look like they should be a single step. Step 4 isn't really necessary; perhaps that, too can be factored into step 2...